<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerFacetedVisualizerModule</class>
 <widget class="qSlicerWidget" name="qSlicerFacetedVisualizerModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="2" colspan="2">
    <widget class="QTabWidget" name="CTKCollapsibleButton">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="CTKCollapsibleButtonPage1">
      <attribute name="title">
       <string>User Query</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>131</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Ontology File Name</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_query">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>45</y>
              <width>111</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Query to Display</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_query">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>90</y>
              <width>114</width>
              <height>32</height>
             </rect>
            </property>
            <property name="text">
             <string>Query!</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>1</y>
              <width>221</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_browse">
            <property name="geometry">
             <rect>
              <x>380</x>
              <y>0</y>
              <width>114</width>
              <height>32</height>
             </rect>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_query">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>50</y>
              <width>221</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
           <widget class="ctkCollapsibleButton" name="CollapsibleButton" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>130</y>
              <width>654</width>
              <height>281</height>
             </rect>
            </property>
            <property name="text" stdset="0">
             <string>Query Results</string>
            </property>
            <widget class="QTreeView" name="treeViewResults">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>30</y>
               <width>351</width>
               <height>211</height>
              </rect>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
             </property>
             <property name="dragEnabled">
              <bool>true</bool>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QPlainTextEdit" name="plainTextEditCommentBox">
             <property name="geometry">
              <rect>
               <x>380</x>
               <y>70</y>
               <width>261</width>
               <height>131</height>
              </rect>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::NoTextInteraction</set>
             </property>
            </widget>
            <widget class="QLabel" name="label_comment">
             <property name="geometry">
              <rect>
               <x>380</x>
               <y>50</y>
               <width>171</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>Comment</string>
             </property>
            </widget>
           </widget>
           <widget class="ctkCollapsibleButton" name="CollapsibleButton_2" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>390</y>
              <width>631</width>
              <height>171</height>
             </rect>
            </property>
            <property name="text" stdset="0">
             <string>QueryLog</string>
            </property>
            <widget class="QTreeView" name="treeViewQueryLog">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>30</y>
               <width>351</width>
               <height>121</height>
              </rect>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Atlas DB Sync</string>
      </attribute>
      <widget class="QLabel" name="label_query_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Atlas model</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_mrml">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_query_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Matching DB atom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_DBatom">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>90</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_mrmlDB">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>114</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
      <widget class="QTreeView" name="treeViewResults_mrmlDB">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>351</width>
         <height>251</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_query_4">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Match atlas model to DB atoms</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
